### PR TITLE
Added details for releasing to a specific platform

### PR DESCRIPTION
**Pull request by ScottHutchinson**
Added instructions for how to release to Linux.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,7 +41,16 @@ The app window will be refreshed when you modify any file in the Renderer projec
...
 
-If you want to specify which platform to release, please take a look at [electron-packager](https://github.com/electron-userland/electron-packager).
+If you want to specify which platform to release, please take a look at [electron-packager](https://github.com/electron-userland/electron-packager). For example, to release to Linux you could do this:
+
+* Add a new `release-linux` script to the package.json file like this:  
+    &nbsp;&nbsp;&nbsp;&nbsp;`"release-linux": "electron-packager . --arch=x64 --platform=linux --out release --overwrite"`
+* Run `yarn run release-linux`
+* Or you could run electron-packager from the command line by first installing npx: `npm install -g npx`
+* Try to run npx from your repo folder by typing `npx -v`
+* If npx is not found, restart Windows to update the Path environment variable to include ..\AppData\Roaming\npm.
```
Comment (MangelMaxime) [inline README.md]: We could write:

- If npx is not found, restart your terminal session to update the Path environment variable

Comment (ScottHutchinson) [reply README.md]: Killing the integrated terminal in VS Code did not work for me. Restarting VS Code did not work for me. Restarting Windows worked.

Comment (MangelMaxime) [reply README.md]: Ah ok... Windows is always fun...

**Comment by MangelMaxime** [comment]
Thank you :)

**Pull request merged**